pandas/tests/io/test_sql.py

```diff
@@ -1374,6 +1390,13 @@ def test_bigint(self):
 
         tm.assert_frame_equal(df, result)
 
+    def test_reading_ints(self):
+        # BigInt should be read as int64, Int should be read as int32, and SmallInt should be read as int16
```

Comment:
Instead of this can you just add the issue number as a comment?

Reply:
@WillAyd this change has been incorporated